### PR TITLE
add 404 and 500 pages

### PR DESCRIPTION
**Pull request by cynthiax**
- Initial 404/500 page
screen shot 2016-09-15 at 11 53 28 am

-- Need future edit on 503 page
-- Need style alignment

### CODE REVIEW

**Comment by stjohnjohnson** [comment]
Oooh, where did we get those images from?

File: app/error/controller.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  errorHeader: 'Don\'t blink',
```
Comment (petey) [inline app/error/controller.js]: Not used by the template.

File: app/error/controller.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  statusCode: 404,
```
Comment (petey) [inline app/error/controller.js]: Should actually be contained in the model passed to the error substate: https://guides.emberjs.com/v2.8.0/routing/loading-and-error-substates/#toc_code-error-code-substates

File: app/error/controller.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  errorMessage: 'Unknown error',
```
Comment (petey) [inline app/error/controller.js]: Also in the model

File: app/router.js [hunk 4]
```diff
@@ -15,6 +15,7 @@ Router.map(function route() {
     this.route('secrets');
   });
   this.route('login');
+  this.route('error', { path: '/*path' });
```
Comment (petey) [inline app/router.js]: I don't know that a route is explicitly needed for an error substate.

File: app/error/controller.js (hunk 1 above)
Comment (cynthiax) [inline app/error/controller.js]: errorHeader was removed

File: app/error/controller.js (hunk 2 above)
Comment (cynthiax) [inline app/error/controller.js]: Discussed with petey, will keep it as default for now

File: app/error/controller.js (hunk 3 above)
Comment (cynthiax) [inline app/error/controller.js]: Discussed with petey, will keep it as default for now

**Pull request merged**